src/main/modules/mpris.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Module in charge of setting and manage mpris service for linux
+   TODO: fetch cover, handle seek bar, handle volume    
+ */
+import * as electron from 'electron';
+import ModuleWindow from './module-window';
+import { TrackModel } from '../../shared/types/interfaces';
+import { fetchCover } from '../../shared/utils';
+import * as mpris  from 'mpris-service';
+const { app, ipcMain } = electron;
+
+class MprisModule extends ModuleWindow {
+
+  player = mpris({
+      name: 'museeks',
+      identity: 'Musseks Music Player',
+      canRaise: true,
+      supportedUriSchemes: ['file'],
+      supportedMimeTypes: ['audio/mpeg', 'application/ogg'],
+      supportedInterfaces: ['player', 'trackList']
```

Comment:
I don't think we support `trackList` do we?

Reply:
That got me thinking, as it doesn't play as long we don't push the play button on the player.
Maybe it needs to add the tracklist for the mpris module to play what's loaded on the player. :confused: